.github/workflows/main.yml

```diff
@@ -48,4 +48,24 @@ jobs:
       - name: Run basic yaml validation
         run: |
           python tests/validate_yaml.py
-        
+  on-fail:
+    if: failure()
+    runs-on: ubuntu-latest
+    needs: check
+    steps:
+      - name: Add label to PR
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const label = 'staging-skip';
+            const { context, github } = require('@actions/github');
+            const owner = context.repo.owner;
+            const repo = context.repo.repo;
+            const pull_number = context.payload.pull_request.number;
```

Comment:
I assume declaring those variables is needed due to the use of `await`? If that's right, then LGTM.

Reply:
On their README seems they declare values as const before await (but sometimes not), so i preferred to be on safe side.